pkg/features/activate.go

```diff
@@ -35,6 +35,10 @@ func Activate(ctx *util.Context) error {
 		return errors.Wrap(err, "failed to install metrics-server")
 	}
 
+	if err := installNodeLocalDNSCache(ctx.Cluster.Features.NodeLocalDNSCache.Enable, ctx); err != nil {
```

Comment:
Is this NPE safe? What if `ctx.Cluster.Features.NodeLocalDNSCache` is nil? Do we check that somewhere before running this?

Reply:
yeah, not it wasn't. now fixed.